waiter/src/waiter/service_description.clj

```diff
@@ -886,11 +888,12 @@
           (if loop-token
             (let [token-data (token->token-data kv-store loop-token token-data-keys true false)]
               (recur (assoc loop-token->token-data loop-token token-data) remaining-tokens))
-            (compute-service-description-template-from-tokens
-              attach-token-defaults-fn token-sequence loop-token->token-data))))
+            (let []
```

Comment:
Can we look at the diff before we open it for review?  Can we minimize unnecessary changes?  Moving functions, etc.

Reply:
Sorry about this `let`, will fix.
Moving `merge-defaults` and `default-and-override` was necessary. Any other you felt moved unnecessarily?

Reply:
Removed the empty `let` block.